src/crewai/cli/authentication/token.py

```diff
@@ -1,9 +1,13 @@
 from .utils import TokenManager
+from logging import getLogger
+
+logger = getLogger(__name__)
 
 
 def get_auth_token() -> str:
     """Get the authentication token."""
     access_token = TokenManager().get_token()
     if not access_token:
-        raise Exception("No token found, make sure you are logged in")
+        logger.warning("No token found, make sure you are logged in")
```

Comment:
@vinibrsl you were the last to touch this. would not raising an exception here break anything ?

Reply:
alternatively i can do a return type of str | None and if None we can handle it

Reply:
Good call. Not raising an exception here would break things like [this](https://github.com/crewAIInc/crewAI/blob/07f8e7395853759bb5a60705e250b4efa29f23aa/src/crewai/cli/command.py#L22). We expect that exception to prompt the user to log in every time `PlusAPI` is instantiated.

Another option would be to raise an specific exception type, and catch it where you don't want to break.